### PR TITLE
-V option added

### LINKED ISSUE #369: CLI docs missing `-v, --version` option description

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
Closes #369

### CODE REVIEW

**Comment by MargaritaLoseva** [comment]
//cc @inikulin @VasilyStrelyaev

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 1]
```diff
@@ -87,6 +87,14 @@ Displays usage information for commands.
 testcafe --help
 ```
 
+### -V, --version
```
Comment (inikulin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: It's `-v` not `-V`

**Comment by inikulin** [comment]
\r-

**Comment by MargaritaLoseva** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
@MargaritaLoseva It requires rebase

**Pull request closed without merging**